samples/Directory.Build.props

```diff
@@ -21,4 +21,10 @@
     <PackageReference Include="System.Net.Http" Version="$(SystemNetHttpVersion)" PrivateAssets="all" ExcludeAssets="all" />
   </ItemGroup>
 
+  <!-- Because of https://github.com/microsoft/vstest/pull/2192 and https://github.com/microsoft/vstest/pull/2067,
+    We can't udpate the the Test SDK version, so explictly referencing the safe Newtonsoft.Json version here -->
+  <ItemGroup Condition="$(IsTestProject)">
```

Comment:
This ensures the test projects under the sample folder are referencing the correct packages. (There are non-test projects under the sample folder)
The `IsTestProject` check is not 100% correct. if the test project is created in the legacy way (e.g. if it is a non-sdk style project, and it doesn't reference the Microsoft.NET.Test.Sdk) it won't work.
However, I don't feel it would apply to any current/future sample test projects.